test/core/python/utility/vector_converter.py

```diff
@@ -18,8 +18,11 @@ def main():
              (np.int64,      "std::int64_t"),
              (np.uint64,     "std::uint64_t"),
              (np.float32,    "float"),
-             (np.float64,    "double"),
-             (np.float128,   "long double")]
+             (np.float64,    "double")]
+
+    # Some architectures (e.g., ARM) do not support float128.
+    if hasattr(np, 'float128'):
+        types.append((np.float128, "long double"))
```

Comment:
What about extending this to all types? I don't know what arch we'll use and which types will be supported

Reply:
I would rather keep it this way to make sure that `float`s and `doubles` and `int`s and `long`s work as expected. Other tests need it anyway. `float128` is a bit nonstandard as it is not even listed in the default list of [numpy data types](https://docs.scipy.org/doc/numpy-1.15.0/user/basics.types.html). Numpy should support those on all non-exotic architectures.

Reply:
fair enough